.github/labeler.yml

```
@@ -1,3 +1,19 @@
 infra:
   - changed-files:
-    - any-glob-to-any-file: ['infra/**']
+      - any-glob-to-any-file: ["infra/**"]
+
+user:
+  - changed-files:
+      - any-glob-to-any-file: ["apps/user-func/openapi.yaml"]
+
+relying-party:
+  - changed-files:
+      - any-glob-to-any-file: ["apps/rp-func/openapi.yaml"]
```

I would also include `apps/rp-func/**` and `apps/user-func/**`

And I suggest you to create a `breaking-change` label for `openapi.yaml` and `asyncapi.yaml` changes (that kind of changes are breaking because can effect the consumers)

```suggestion
user:
  - changed-files:
      - any-glob-to-any-file: ["apps/user-func/**"]

relying-party:
  - changed-files:
      - any-glob-to-any-file: ["apps/rp-func/**"]
```

---

Nice suggestion, please take a look here for the new label [af7ae7a](https://github.com/pagopa/io-fims/pull/42/commits/af7ae7ac7452267ac4a598151959c05f360d10ee)